src/options.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * These are all options available to Ars Arsenal
+ * @flow
+ */
```

Comment:
I love all the comments in this file. It's really easy to follow and see how this would be used.